yt/fields/particle_fields.py

```diff
@@ -586,26 +568,16 @@ def _particle_velocity_spherical_radius(field, data):
         validators=[ValidateParameter("normal"), ValidateParameter("center")],
     )
 
-    def _particle_spherical_velocity_radius(field, data):
-        """This field is deprecated and will be removed in a future release"""
-        return data[ptype, "particle_velocity_spherical_radius"]
-
-    registry.add_field(
+    registry.alias(
         (ptype, "particle_spherical_velocity_radius"),
-        sampling_type="particle",
-        function=_particle_spherical_velocity_radius,
-        units=unit_system["velocity"],
-        validators=[ValidateParameter("normal"), ValidateParameter("center")],
+        (ptype, "particle_velocity_spherical_radius"),
+        deprecate=("4.0.0", "4.1.0"),
     )
 
-    # particel_velocity_spherical_radius is simply aliased to
-    # "particle_radial_velocity" for convenience
-    registry.add_field(
+    registry.alias(
         (ptype, "particle_radial_velocity"),
-        sampling_type="particle",
-        function=_particle_spherical_velocity_radius,
-        units=unit_system["velocity"],
-        validators=[ValidateParameter("normal"), ValidateParameter("center")],
+        (ptype, "particle_velocity_spherical_radius"),
```

Comment:
Do we want to deprecate here? It seems like that wasn't the intent originally?

Reply:
You're right, will fix